### DISCUSSION TITLE
Wind direction & speed

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1063] [General]
Hello
Cannot get neither wind direction nor  speed in tour info from Garmin Edge 1030 GPS
Can I get those from either web meteo sites within MTB?

Thank you

**Comment by FJBDev** [comment]
@gringojack Please read the documentation where it mentions how to retrieve the weather [here](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/documentation-tools-weather). Note that the page is not up-to-date and MT now offers 3 weather providers as of 23.3.2

![image](https://user-images.githubusercontent.com/25235665/231833154-7e2ae6fe-c62a-4467-ab52-3e4922f835e6.png)

**Reply by gringojack** [reply]
Thank you much
